modules/ROOT/pages/kubernetes/authentication-authorization.adoc

```diff
@@ -0,0 +1,92 @@
+[[configure-authentication-and-authorization]]
+= Authentication and authorization
```

Comment:
might want to replace authorization with Authorization

Reply:
No, we use sentence case for titles. If you've seen anywhere capital case, it's a mistake.